stable_diffusion/stable_diffusion.ipynb

```diff
@@ -218,45 +221,46 @@
     "class StableDiffusion(param.Parameterized):\n",
     "    prompt = param.String(doc=\"\"\"\n",
     "        Text describing the image you wish to generate\"\"\")\n",
-    "    \n",
+    "\n",
     "    negative_prompt = param.String(doc=\"\"\"\n",
     "        Text describing what _not_ to include in the image (for refining results)\"\"\")\n",
-    "    \n",
+    "\n",
     "    model = param.Selector(objects=list(pipelines), default=default_model, doc=\"\"\"\n",
     "        A pre-trained model to be used for inference\"\"\")\n",
-    "    \n",
+    "\n",
     "    _size_range = tuple(448 + i*2**6 for i in range(10))\n",
     "    width = param.Selector(_size_range, default=_size_range[1], doc=\"\"\"\n",
     "        Width (in pixels) of the images to generate\"\"\")\n",
-    "    \n",
+    "\n",
     "    height = param.Selector(_size_range, default=_size_range[1], doc=\"\"\"\n",
     "        Height (in pixels) of the images to generate\"\"\")\n",
-    "    \n",
+    "\n",
     "    guidance_scale = param.Number(bounds=(5, 10), softbounds=(7, 8.5), step=0.1, default=7.5, doc=\"\"\"\n",
-    "        How closely the model should try to match the prompt, at the \n",
+    "        How closely the model should try to match the prompt, at the\n",
     "        potential expense of image quality or diversity.\n",
     "        Also known as CFG (Classifier-free guidance scale).\"\"\")\n",
-    "    \n",
+    "\n",
     "    num_steps = param.Integer(label='# of steps', bounds=(10, 75), default=30, doc=\"\"\"\n",
-    "        How many denoising steps to take. \n",
+    "        How many denoising steps to take.\n",
     "        More steps takes longer but gives a more-refined image.\"\"\")\n",
     "\n",
     "    seed = param.Integer(label='Random seed', default=1,\n",
     "                         bounds=random_int_range, step=10, precedence=1, doc=\"\"\"\n",
     "        Seed controlling the noise values generated.\"\"\")\n",
-    " \n",
+    "\n",
     "    generate = param.Event(precedence=1)\n",
-    "    \n",
-    "    param.depends(\"generate\")\n",
+    "\n",
+    "    @param.depends(\"generate\")\n",
```

Comment:
@sandhujasmine it seems that there was a missing `@` on this line, which I believe meant that the decorator wasn't applied. We'd better double check what's the correct change, either what I've done or removing entirely this line.

Reply:
That's odd; seems like it wouldn't have worked without that!